sdk/keyvault/keyvault-admin/samples/v4-beta/javascript/README.md

```diff
@@ -0,0 +1,75 @@
+---
+page_type: sample
+languages:
+  - javascript
+products:
+  - azure
+  - azure-key-vault
+urlFragment: keyvault-admin-javascript-beta
+---
+
+# Azure Key Vault Administration client library samples for JavaScript (Beta)
+
+These sample programs show how to use the JavaScript client libraries for Azure Key Vault Administration in some common scenarios.
+
+| **File Name**                                             | **Description**                                                                                       |
+| --------------------------------------------------------- | ----------------------------------------------------------------------------------------------------- |
+| [accessControlHelloWorld.js][accesscontrolhelloworld]     | Uses an AccessControlClient to list, create, and assign roles to users.                               |
+| [backupRestoreHelloWorld.js][backuprestorehelloworld]     | Uses a BackupClient to backup and fully restore an Azure Key Vault using Azure Storage Blob.          |
+| [backupSelectiveKeyRestore.js][backupselectivekeyrestore] | Uses a BackupClient to backup and restore a specific key in Azure Key Vault using Azure Storage Blob. |
```

Comment:
For references here and below: the admin package only pertains to Managed HSM. The keys package can apply (most times) to either KV or MHSM, and secrets and certs are KV-only.